modules/kogito-data-index-oracle/module.yaml

```
@@ -0,0 +1,12 @@
+schema_version: 1
+name: org.kie.kogito.dataindex.oracle
+version: "2.0.0-snapshot"
+
+artifacts:
+  - name: data-index-service-oracle-runner.jar
+    url: https://repository.jboss.org/nexus/content/groups/public/org/kie/kogito/data-index-service-oracle/2.0.0-SNAPSHOT/data-index-service-oracle-2.0.0-SNAPSHOT-runner.jar
```

Please use a specific snapshot version here.

```suggestion
    url: https://repository.jboss.org/nexus/content/groups/public/org/kie/kogito/data-index-service-oracle/2.0.0-SNAPSHOT/data-index-service-oracle-2.0.0-20220124.043203-5-runner.jar
```